src/bitpay.merchant.php

```diff
@@ -305,20 +294,28 @@ function form_bitpay()
         create_table();
 
         // Protect your data!
-        $mcrypt_ext  = new \Bitpay\Crypto\McryptExtension();
-        $fingerprint = substr(sha1(sha1(__DIR__)), 0, 24);
+        $mcrypt_ext   = new \Bitpay\Crypto\McryptExtension();
+        $fingerprint  = substr(sha1(sha1(__DIR__)), 0, 24);
+        $bitpayjsfile = plugins_url('/bitpay/assets/js/bitpay.js', __FILE__);
 
-        // Load Script onto settings page
-        $load_script = file_get_contents(plugins_url('/bitpay/assets/js/bitpay.js', __FILE__));
-        $script = '<script type="text/javascript">'. $load_script.'</script>';
+        if (true === file_exists($bitpayjsfile) &&
```

Comment:
$bitpayjsfile is set to a URL in line 299 via plugins_url.
file_exists is thus being called on a URL resource instead of a file resource and causing an error